src/main/java/seedu/duke/financial/Expense.java

```diff
@@ -26,7 +26,12 @@ public Expense(double amount, String description, LocalDate date) {
      */
     @Override
     public String toString() {
-        DateTimeFormatter pattern = DateTimeFormatter.ofPattern("dd/MM/yy");
+        DateTimeFormatter pattern = DateTimeFormatter.ofPattern("yyyy-MM-dd");
```

Comment:
Is it possible to stick to the original format of dd/MM/yy?

Reply:
done